src/vs/workbench/browser/parts/auxiliarybar/auxiliaryBarPart.ts

```diff
@@ -126,63 +126,3 @@ export class AuxiliaryBarPart extends BasePanelPart {
 		};
 	}
 }
-
```

Comment:
Were the css changes also supported to be in this pr?

Reply:
yea the title got messed up due to the latest commit